src/components/BnFileInput/BnFileInput.vue

```diff
@@ -166,6 +177,10 @@ function removeFile(file: File) {
         },
         props.classes.wrapper,
       ]"
+      :tabindex="props.disabled || hasDefaultSlot || isAvatar ? '1' : '0'"
+      @blur="handleBlur"
```

Comment:
`handleBlur` doesn't seem to emit anything. I think you'll have to add a wrapper function that triggers `handleBlur` and emits the event.

https://github.com/platanus/banano/assets/472791/5095a092-cd8e-4834-ab43-33dac243a24d

Also, since it's included by default, I think the avatar template should still trigger focus/blur events. 🤔

Reply:
I realized that `handleBlur` was setting vee-validate `setTouched`, so I deleted it from `OpenDialog` function. That's why some `BnFileInput` tests had to change.